apps/src/javalab/JavalabButton.jsx

```diff
@@ -64,19 +65,9 @@ JavalabButton.defaultProps = {
 };
 
 const styles = {
-  button: {
-    display: 'flex',
-    flexDirection: 'column',
-    alignItems: 'center',
-    color: color.white
-  },
-  horizontal: {
-    flexDirection: 'row'
-  },
   disabled: {
     backgroundColor: color.light_gray,
     borderColor: color.light_gray,
-    ':hover': null,
     cursor: 'default'
   },
   verticalPadding: {
```

Comment:
Could this `textStyle` also be moved into CSS?

Reply:
I swear I looked at this, but I can't see why not! I'll take a look.